validation/src/main/scala/hmda/validation/engine/ts/validity/TsValidityEngine.scala

```diff
@@ -2,29 +2,25 @@ package hmda.validation.engine.ts.validity
 
 import hmda.model.fi.ts.TransmittalSheet
 import hmda.validation.api.ValidationApi
+import hmda.validation.engine.ValidationError
 import hmda.validation.engine.ts.TsCommonEngine
+import hmda.validation.rules.EditCheck
 import hmda.validation.rules.ts.validity.{ V105, V140, V155 }
 
-trait TsValidityEngine extends TsCommonEngine with ValidationApi {
-
-  private def v105(t: TransmittalSheet): TsValidation = {
-    convertResult(t, V105(t), "V105")
-  }
+import scalaz._
 
-  private def v140(t: TransmittalSheet): TsValidation = {
-    convertResult(t, V140(t), "V140")
-  }
+trait TsValidityEngine extends TsCommonEngine with ValidationApi {
 
-  private def v155(t: TransmittalSheet): TsValidation = {
-    convertResult(t, V155(t), "V155")
+  private def doCheck[T](editCheck: EditCheck[T], input: T): ValidationNel[ValidationError, T] = {
```

Comment:
Mostly because it was the end of the day/week in my timezone. This is definitely some of the duplication I mentioned. I'm happy to go ahead and fix it if frees things up to let merging happen. (But these lines are pretty far from the ones that are likely to cause conflicts and such, so I could see it going either way.)